documentation/getting-started/getting-started-project-wizard.asciidoc

```diff
@@ -0,0 +1,70 @@
+---
+title: Creating a Project with the Project Wizard
+order: 10
+layout: page
+---
+
+[[getting-started.project-wizard]]
+= Creating a Project with the Project Wizard
+
+== Creating a Downloadable Project
+
+In the following, we describe how you can create a downloadable project with the project wizard.
+
+. Navigate to http://vaadin.com/start[vaadin.com/start].
+
+. Choose the Vaadin version for the project.
+The latest LTS (Long-Term Service) version is suggested, but you can also choose a newer sversion if one is available.
++
+.Choosing the Vaadin version.
+image::images/project-wizard-01.png[Project Wizard]
+
+. Select *Download* (default)
+
+. Select a technology stack.
++
+See the descriptions of the technology stacks in <<getting-started-overview#getting-started.overview.stacks, Technology Stacks>>.
++
+image::images/project-wizard-02.png[Download]
+
+. Enter a *Maven Group ID*.
+This is usually your corporation ID or a personal ID.
+You are welcome to use one under `org.vaadin`, especially for open-source projects.
+
+. Enter a *Project Name*.
+It will also be used for the project ID in lowercase, with spaces replaced by dashes.
+
+. Click *Download* _or_ proceed to customize it as described in <<#getting-started.project-wizard.customizing, Customizing a Project>>.
```

Comment:
Going to the wizard from here does not preserve any of the choices the user has made so far regarding stack or project name etc. Mentioning customize at this step is probably not a good idea, it should probably be after this list as a side note.